landing-page/src/components/landing-page-logs.ts

```diff
@@ -216,7 +220,12 @@ class LandingPageLogs extends LitElement {
       // eslint-disable-next-line no-console
       console.error(err);
 
-      // fallback to observerlogs if there is a problem with supervisor
+      if (!retry) {
```

Comment:
if retry is false, we are gonna start it again? That sounds like a retry?

Reply:
retry is true when it is retried and I want to retry it just once. Maybe I can rename it to something clearer